OrbitGl/TrackAccessibility.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TrackAccessibility.h"
+
+#include "GlCanvas.h"
+#include "TimeGraph.h"
+#include "Track.h"
+
+namespace orbit_gl {
+
+using orbit_accessibility::AccessibleInterface;
+
+const AccessibleInterface* AccessibleTrackContent::AccessibleParent() const {
+  return track_->AccessibilityInterface();
+}
+
+std::string AccessibleTrackContent::AccessibleName() const {
+  CHECK(track_ != nullptr);
+  return track_->GetName() + "_content";
+}
+
+AccessibilityRect AccessibleTrackContent::AccessibleLocalRect() const {
+  CHECK(track_ != nullptr);
+  CHECK(track_->GetTimeGraph() != nullptr);
+
+  auto& layout = track_->GetTimeGraph()->GetLayout();
+  return AccessibilityRect(0, static_cast<int>(layout.GetTrackTabHeight()),
+                           static_cast<int>(track_->GetSize()[0]),
+                           static_cast<int>(track_->GetSize()[1]));
+}
+
+AccessibilityState AccessibleTrackContent::AccessibleState() const {
+  AccessibilityState result;
+  return result;
+}
+
+const AccessibleInterface* AccessibleTrackTab::AccessibleParent() const {
+  return track_->AccessibilityInterface();
+}
+
+std::string AccessibleTrackTab::AccessibleName() const {
+  CHECK(track_ != nullptr);
+  return track_->GetName() + "_tab";
+}
+
+AccessibilityRect AccessibleTrackTab::AccessibleLocalRect() const {
+  CHECK(track_ != nullptr);
+  CHECK(track_->GetTimeGraph() != nullptr);
+
+  auto& layout = track_->GetTimeGraph()->GetLayout();
+  return AccessibilityRect(static_cast<int>(layout.GetTrackTabOffset()), 0,
+                           static_cast<int>(layout.GetTrackTabWidth()),
+                           static_cast<int>(layout.GetTrackTabHeight()));
```

Comment:
Similarly to previous comment, I feel that we are duplicating the track layout logic in different classes. If a track diverges from the common pattern in the future, then we'll break this method and the accessible rectangle will be out of sync with the new track.